### ISSUE TITLE
Anaconda screws up Windows OS

### ISSUE THREAD

**Issue by labkw** [conda/conda#8976]
**Anaconda version**: 4.6.15

**Expected behavior**: When you double-click an executable file in Windows, the application will run as expected. When you double-click a non-executable file, a suitable application will start or come front with that file opened for you.

**Actual behavior**: A multitude of applications will at some point crash out with error report:
```
Traceback ():
  ...
  File "...\anaconda\lib\site-packages\<any package>\_distributor_init.py", line ##, in <module>
    from . import _mklinit
ImportError: DLL load failed: The specified module could not be found.
```

**Applications affected**: Google Chrome, Microsoft Visual Studio, SublimeText, several games; and in general any application that at some point of their lifetime try to do something pythonic.

**Work around**: Create a .BAT script file, for each and every installed application you will ever want to use, because you'll never know if they need python or not, as follows (example):
```
conda activate base
START "" "C:\Program Files (x86)\Microsoft Visual Studio\2017\Enterprise\Common7\IDE\devenv.exe"
```
Then painstakingly open the desired file from the file menu of said application... 

The cause of this issue is known: it requires that some human agent activates an environment, in each and every process that at some point requires the use of python; or at some point may spawn a sub-process that will do that.

The issue has been reported already in multiple github issues, invariably dismissed and closed by Anaconda as "feature no bug". It may be that Anaconda, Inc. has designed its products to burden its clients in such a way but IMO a flaw in a design is as much of bug as an error in its execution would be, so I would appreciate a forthcoming fix.

**Comment by msarahan** [comment]
In those multiple GitHub issues, you have surely encountered the link to our troubleshooting page, where the reason for this is explained and workarounds are suggested. It seems like you probably want to set the CONDA_DLL_SEARCH_MODIFICATION_ENABLE env var. This is not really a feature, so much as it is a technical fact that must be dealt with. It is just part of running on Windows with its library search behavior.

In case you missed it: https://conda.io/projects/conda/en/latest/user-guide/troubleshooting.html#numpy-mkl-library-load-failed

**Comment by labkw** [comment]
Thank you for closing the issue. Needless to say, none of the suggested workarounds fix the problem. I will proceed to uninstall Anaconda and advise my organisation to steer away from it.

**Comment by msarahan** [comment]
Since you made no effort to say how the workarounds fail to address your issue, yes, we're at a loss here.  Good luck.

**Comment by labkw** [comment]
Well, I've tried all the workarounds listed on the support page you linked to: update python, remove "MKL related files" from sys32 and set four environment variables. In fact, I did all before posting this issue. To be frank, I'm not sure what an "MKL related file" exactly is supposed to be but I searched for files with names matching MKL and there weren't any.

The point is the first sentence on the support page: **"If you are not activating your environments, start with doing that."** If you read my OP, the issue is not that the activation does not work, or numpy won't load after activation. The issue is that Anaconda requires that end users, our clients, need to teleport back to the 1980's to fiddle on a command line. To them, that would be an unacceptable deal breaking requirement for any system. If in addition, installation of Anaconda breaks applications like Google Chrome  on their system, we'll have a liability we cannot afford.

HTH
